django_eth_events/utils.py

```diff
@@ -42,7 +42,10 @@ def normalize_address_without_0x(address) -> str:
 class JsonBytesEncoder(JSONEncoder):
     def default(self, obj):
         if isinstance(obj, bytes):
-            return '0x' + obj.hex()
+            try:
+                return obj.decode()
+            except UnicodeDecodeError:
+                return '0x' + obj.hex()
```

Comment:
Change this line for `HexBytes(obj).hex()`, so `.hex()` will always have `0x`

Reply:
top solution ;)